docs/exception-mandate-for-tenant-level-api-path.md

```diff
@@ -0,0 +1,44 @@
+# ExceptionMandateForTenantLevelApiPath
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Uri-V1-11
+
+## Description
+
+Tenant level API's are not allowed for a PUT operation. Exception from PAS team is mandatory for the very first implementation. 
```

Comment:
Tenant level APIs are strongly discouraged and subscription or resource group level APIs are preferred instead. If you cannot model your APIs at these levels, you will need to present your design and get an exception from the PAS team.